docs/ops/cli.md

```diff
@@ -170,29 +170,13 @@ These examples about how to manage a job in CLI.
 
         ./bin/flink cancel <jobID>
 
--   Cancel a job with a savepoint:
+-   Cancel a job with a savepoint (deprecated; use "stop" instead):
 
         ./bin/flink cancel -s [targetDirectory] <jobID>
 
--   Stop a job with a savepoint (streaming jobs only):
+-   Gracefully stop a job with a savepoint (streaming jobs only):
```

Comment:
suggest to change to `Gracefully stop a job, optionally with a savepoint (streaming jobs only)`

Reply:
It rather keep it as it is. The `stop` command right now always performs a savepoint. It's not optional. 

I understand, that the stop command in the future might or might not take a savepoint, but the documentation should reflect the current state.

Reply:
Makes sense.